src/main/java/seedu/dietbook/calculator/Calculator.java

```diff
@@ -0,0 +1,297 @@
+package seedu.dietbook.calculator;
```

Comment:
Recommend splitting the class/avoiding a do-it-all class if you find this applicable.